ui/helpers/utils/multichain/blockExplorer.test.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  MULTICHAIN_PROVIDER_CONFIGS,
+  MultichainNetworks,
+} from '../../../../shared/constants/multichain/networks';
+import {
+  CHAIN_IDS,
+  MAINNET_DISPLAY_NAME,
+  CHAIN_ID_TO_RPC_URL_MAP,
+  ETH_TOKEN_IMAGE_URL,
+  NETWORK_TYPES,
+  CURRENCY_SYMBOLS,
+} from '../../../../shared/constants/network';
+import { MultichainNetwork } from '../../../selectors/multichain';
+import {
+  getMultichainBlockExplorerUrl,
+  getMultichainAccountUrl,
+} from './blockExplorer';
+
+const mockEvmNetwork: MultichainNetwork = {
+  nickname: 'Ethereum',
+  isEvmNetwork: true,
+  chainId: 'eip155:1',
+  network: {
+    chainId: CHAIN_IDS.MAINNET,
+    nickname: MAINNET_DISPLAY_NAME,
+    rpcUrl: CHAIN_ID_TO_RPC_URL_MAP[CHAIN_IDS.MAINNET],
+    rpcPrefs: {
+      imageUrl: ETH_TOKEN_IMAGE_URL,
+      blockExplorerUrl: 'https://etherscan.io',
+    },
+    type: NETWORK_TYPES.MAINNET,
+    ticker: CURRENCY_SYMBOLS.ETH,
+    id: NETWORK_TYPES.MAINNET,
+  },
+};
+
+const mockNonEvmNetwork: MultichainNetwork = {
+  nickname: 'Bitcoin',
+  isEvmNetwork: false,
+  chainId: MultichainNetworks.BITCOIN,
+  network: MULTICHAIN_PROVIDER_CONFIGS[MultichainNetworks.BITCOIN],
+};
+
+describe('Block Explorer Tests', () => {
+  describe('getMultichainBlockExplorerUrl', () => {
+    it('should return the correct block explorer URL for Ethereum mainnet', () => {
```

Comment:
Let's get rid of the "should" here (to align with what the contributor docs recommendations)